lib/ts/utils/windowHandler/index.ts

```diff
@@ -12,38 +12,45 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
-import { defaultWindowHandlerImplementation } from "./defaultImplementation";
-import { WindowHandlerInterface, WindowHandlerInput } from "./types";
+import { defaultWindowHandlerImplementation } from './defaultImplementation'
+import { WindowHandlerInterface, WindowHandlerInput } from './types'
 
 export default class WindowHandlerReference {
-    private static instance?: WindowHandlerReference;
-
-    windowHandler: WindowHandlerInterface;
+    private static instance?: WindowHandlerReference
+
+    // Declare subset of WindowHandlerReference that is actually used,
+    // so implementors don't need to over-engineer.
+    windowHandler: {
+        location: Pick<
+            WindowHandlerInterface['location'],
+            'getHostName' | 'getOrigin'
+        >
+    }
 
     constructor(windowHandlerInput?: WindowHandlerInput) {
-        let windowHandlerFunc: WindowHandlerInput = original => original;
+        let windowHandlerFunc: WindowHandlerInput = original => original
```

Comment:
These updates do not match the coding style used elsewhere in the project; please run `npm run build-pretty`.
Also, I'm not sure if you've installed the commit hooks (check [here](https://github.com/supertokens/supertokens-website/blob/master/CONTRIBUTING.md#development-setup) if not)

Reply:
<img width="319" alt="image" src="https://user-images.githubusercontent.com/525211/212615170-6f15c569-a5db-44e8-912a-2976dec81cd9.png">

I think prettier might need to be updated ? Seems this is using 1.x while 2.x is out.